excel/clean.py

```diff
@@ -44,7 +44,8 @@ def entity_has_attribute(object_data: dict, attribute: str) -> bool:
 
 
 def value_populated(value: str) -> bool:
```

Comment:
Could we rename it `is_value_populated`, please?
And use it accordingly.

Thanks

Reply:
change made